test/e2e/common/util_cleanup.go

```diff
@@ -149,9 +225,9 @@ func DeleteAllPools() {
 		logf.Log.Info("DeleteAllPools: deleting MayastorPools")
 		for _, pool := range pools.Items {
 			logf.Log.Info("DeleteAllPools: deleting", "pool", pool.GetName())
-			err = gTestEnv.DynamicClient.Resource(poolGVR).Namespace("mayastor").Delete(context.TODO(), pool.GetName(), metav1.DeleteOptions{})
+			err = gTestEnv.DynamicClient.Resource(poolGVR).Namespace("mayastor").Delete(context.TODO(), pool.GetName(), metav1.DeleteOptions{GracePeriodSeconds: &ZeroInt64})
```

Comment:
Use constant string for mayastor namespace (as elsewhere) ?

Reply:
I was deferring that as part of the refactor and cleanup change which is in the pipeline.